src/components/common/CaseDetails/CaseDetails.js

```diff
@@ -188,28 +233,20 @@ function CaseDetails() {
                   </div>
                 );
               })
-            : placeholderArray.map(item => {
+            : notes.map(item => {
+                let date = new Date(item.created_at);
```

Comment:
I see you are grabbing the month/date/year.  Did that want a time when this was added?  If so, you can format the date using new Date(item.created_at).toLocaleString()

Reply:
If not you could still clean it up a little bit using toLocaleDateString() instead